jscomp/ml/typecore.ml

```diff
@@ -884,6 +913,20 @@ module Label = NameChoice (struct
   type t = label_description
   let type_kind = "record"
   let get_name lbl = lbl.lbl_name
+  
+  let add_with_name lbl name =
```

Comment:
perhaps a more scary sounding name rather than just having a warning in the comment

Reply:
Yes, good point!